docs/modules/ROOT/pages/explanations/decisions/scheduled-mr-merges.adoc

```diff
@@ -0,0 +1,63 @@
+= Scheduling the merge of a GitLab merge request
+
+== Problem
+
+We want to be able to merge GitLab merge requests at preset times.
+This enables us to roll out changes during maintenance windows in an automated fashion.
```

Comment:
I'd add a short paragraph here noting that while there's a GitLab feature request for scheduled merges, there's currently no suitable functionality out of the box.